CHANGELOG.unreleased.md

```diff
@@ -44,6 +44,115 @@ const link = <a href="example.com">http://example.com</a>;
 
 -->
 
+#### TypeScript: Support TypeScript 3.7 syntax features ([#6657] by [@cryrivers])
+
+Add following syntax support for TypeScript 3.7:
+
+- Optional Chaining
+- Null Coalescing
+- `asserts` Keyword
+- `declare` keyword on class members
+
+##### Optional Chaining
+
+Previous versions would not be able to format this syntax, this has been fixed in this version.
```

Comment:
This description is not needed. The title suffices for the description. (Please see the comments in the beginning of `CHANGELOG.unreleased.md`.)